### PR TITLE
CSS Brochure Form - Michael Van Doorn

### PR DESCRIPTION
**Pull request by Michaelvxd**
CleanShot 2022-12-15 at 12 46 29@2x

### CODE REVIEW

File: css-brochure-form/styles.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  background-color: rgb(238 85 54);
+  color: white;
+}
+
+#questions {
+  margin: 10px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
```
Comment (scrumptiousdev) [inline css-brochure-form/styles.css:56]: Like I mentioned during previous Q&A session, we should avoid styling element with id

File: css-brochure-form/index.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  <div>
+    <p class="learn"> LEARN MORE </p>
+  </div>
+  <div>
+    <form id="questions">
+      <div >
+        <div class="question-prompt">
+          <label for="email" class="labeltxt">COMPANY EMAIL</label>
+          <div>
+            <input type="email" name="email" placeholder="sally@company.co" required>
```
Comment (scrumptiousdev) [inline css-brochure-form/index.html:24]: Important attributes are missing for all inputs / fields in order to connect with labels

**Review by scrumptiousdev — CHANGES_REQUESTED** [review]

File: css-brochure-form/index.html (hunk 2 above)
Comment (Michaelvxd) [reply css-brochure-form/index.html]: Updated!

File: css-brochure-form/styles.css (hunk 1 above)
Comment (Michaelvxd) [reply css-brochure-form/styles.css:56]: Updated!

**Review by scrumptiousdev — APPROVED** [review]
Approved!

**Pull request merged**